src/providers/cloudstack/configdrive.rs

```diff
@@ -84,7 +84,7 @@ impl ConfigDrive {
     }
 
     fn mount_ro(source: &Path, target: &Path, fstype: &str) -> Result<()> {
-        mount::mount(Some(source), target, Some(fstype), mount::MS_RDONLY, None::<&str>)
+        mount::mount(Some(source), target, Some(fstype), mount::MsFlags::MS_RDONLY, None::<&str>)
```

Comment:
Did this mean to be put into this commit?

Reply:
Yes, it's due to an API change in `nix`.

Reply:
Yeah, looks like the constants were moved from 0.9 to 0.11.

Reply:
Ahh, too slow. :)